### ISSUE TITLE
Capture "Ctrl + Enter" or "⌘ + Enter" to send SQL query?

### ISSUE THREAD

**Issue by rayvoelker** [simonw/datasette#1253]
It appears as though "Shift + Enter" triggers the form submit action to submit SQL, but could that action be bound to the "Ctrl + Enter" or "⌘ + Enter" action?

I feel like that pattern already exists in a number of similar tools and could improve usability of the editor.

**Comment by dufferzafar** [comment]
Yeah, I was pressing Ctrl + Enter as well. Came here to open this issue and found out Shift + Enter works.

@simonw Any way to configure this?
